frontend/src/modules/LoadTestContentRedux.js

```diff
@@ -34,10 +35,28 @@ const getTestInstructions = testName => {
   };
 };
 
+const updateTestQuestionsState = testQuestions => ({
+  type: UPDATE_TEST_QUESTIONS,
+  testQuestions
+});
+
+const getTestQuestions = testName => {
+  return async function() {
+    let testQuestionsContent = await fetch(`/api/test-questions/?test_name=${testName}`, {
+      method: "GET",
+      Accept: "application/json",
+      "Content-Type": "application/json",
+      cache: "default"
+    });
+    return await testQuestionsContent.json();
+  };
+};
+
 // Initial State
 const initialState = {
   testMetaData: "",
-  testInstructions: ""
+  testInstructions: "",
```

Comment:
I think these should also start as objects.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/356/commits/501c8b119c0fbd7e1c4d1faa1867e83e8ddd4192